### ISSUE TITLE
[Screen Reader-M365Admin Portal-Service health] : Screen reader does not announce role for 'Export' control.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13201]
### Steps to reproduce:
1.	Open URL and login with valid credentials.
2.	Navigate to table data in 'All Issues' tab and activated through enter key.
3.	Navigate to table list item and activate it.
4.	Start NVDA and navigate to Export control in 'Tenant affected' tab by tab key.
5.	Verify that screen reader does not announce role for 'Export' control or not.

### Actual behavior:
Screen reader does not announce role for 'Export' control. Screen reader announces only 'Export'.

### Expected behavior:
Screen reader should announce role for 'Export' control. 

### System configuration
Application: M365 Admin Portal
URL: https://admin.microsoft.com/Adminportal/Home
Browser: Edge, Version 96.0.1054.62 (Official build) (64-bit)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA Version 2021.1

#### Windows version:
Window 11, Version 21H2 (OS Build 22000.376)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, also repro with NVDA 2020.4

**Comment by feerrenrut** [comment]
This needs further investigation by Microsoft developers. Please provide a minimal test case for us to investigate. I'm closing this issue for now.
